src/test/java/bio/terra/integration/DataRepoClient.java

```diff
@@ -166,14 +167,14 @@ private String getLocationHeader(DataRepoResponse<JobModel> jobModelResponse) {
 
         if (response.getStatusCode().is2xxSuccessful()) {
             if (responseClass != null) {
-                T responseObject = objectMapper.readValue(response.getBody(), responseClass);
```

Comment:
Since the `objectMapper.readValue` pattern is in a few other files, perhaps it might be useful to reuse `mapFromJson` for it?

Reply:
I was trying to do something simple and quick to help with debugging these ephemeral failures.
How about I make a ticket to do the more general fix?

Reply:
Sounds good.